### DISCUSSION TITLE
How to find modified site's location?

### DISCUSSION THREAD

**Discussion by Munchic** [vdemichev/DiaNN#434] [Q&A]
Hello,

I was wondering if there is an easy way to get phosphosite location on a protein from the Modified.Sequences that DIA-NN outputs? For example, RSD format (e.g., S5-p) or trailing 7AA format (___MAAAsPLRDCQA).

Thank you for your help!

**Comment by vdemichev** [comment]
Hi,

Please try https://github.com/MannLabs/alphamap 

Best,
Vadim

**Reply by Munchic** [reply]
Thank you!

**Comment by Munchic** [comment]
Hi Vadim,

I wanted to follow-up on this. Would it be possible to add a column in DIA-NN's report specifying the modified site location(s) on the protein? E.g., separated by `;` like in MaxQuant if there are multiple modified sites or protein isoforms.

The AlphaMap tool seems to query per protein but I need modified site locations for all ID'ed peptides from DIA-NN.

Thank you!

Munchic

**Reply by vdemichev** [reply]
Potentially yes, but this is not planned for the next months.

**Reply by Munchic** [reply]
Okay, thank you!
